### ISSUE TITLE
CustomResourceDefinition is deprecated in v1.16+

### ISSUE THREAD

**Issue by sparkylarky** [kubeapps/kubeapps#2884]
### Description:

Trying to install kubeapps using the recommended approach but receive the error message "apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition".

Kubeapps is running and can be accessed, however the notification "Unable to retrieve API info: service unavailable" shows.

### Steps to reproduce the issue:

<pre>$ kubectl create namespace kubeapps
namespace/kubeapps created</pre>

<pre>$ helm install kubeapps --namespace kubeapps bitnami/kubeapps
W0525 15:54:06.243914 1401919 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
NAME: kubeapps
LAST DEPLOYED: Tue May 25 15:54:06 2021
NAMESPACE: kubeapps
STATUS: deployed
REVISION: 1
NOTES:
** Please be patient while the chart is being deployed **
...
</pre>
### Version of Helm, Kubeapps and Kubernetes:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.5.4", GitCommit:"1b5edb69df3d3a08df77c9902dc17af864ff05d1", GitTreeState:"clean", GoVersion:"go1.15.11"}
```

- Output of `helm list -n <kubeapps-namespace> <kubeapps-release-name>`:

```
NAME    	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART         	APP VERSION
kubeapps	kubeapps 	1       	2021-05-25 15:54:06.460463335 +0000 UTC	deployed	kubeapps-7.0.3	2.3.2
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.7-34+df7df22a741dbc", GitCommit:"df7df22a741dbc18dc3de3000b2393a1e3c32d36", GitTreeState:"clean", BuildDate:"2021-05-12T21:08:20Z", GoVersion:"go1.15.10", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.7-34+df7df22a741dbc", GitCommit:"df7df22a741dbc18dc3de3000b2393a1e3c32d36", GitTreeState:"clean", BuildDate:"2021-05-12T21:09:51Z", GoVersion:"go1.15.10", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by antgamdia** [comment]
Hi, let's try to discover what's happening here :)

The warning message is related to #2169 , but it is not a problem unless you are in k8s >1.21 (from memory), but that's not your case.

> Kubeapps is running and can be accessed, however the notification "Unable to retrieve API info: service unavailable" shows

What do you mean by running? Are you able to deploy something or use the catalog?
Could you share a screenshot with the network tab of your browser dev console?

**Comment by sparkylarky** [comment]
Thank you for helping out, antgamdia. I was about to take a screenshot and noticed that Kubeapps is working like a charm.
I'm not entirely sure how I got it working, but I updated my Helm repo and changed the permission of the file "config".

<pre>
$ chmod 700 /home/user/.kube/config
</pre>

**Comment by antgamdia** [comment]
Happy to hear! I don't know which values.yaml are you using, but we released a Chart v7.0.4 this morning with a minor bug fix (https://github.com/bitnami/charts/commit/6e1439825a53e57b08bb85689c481e83384415de). But this is unrelated to your issue. so perhaps it was just a transient problem (lack of cpu/mem resources maybe?).

Closing, but feel free to reopen it or to file another issue if you face any problems :)
